src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java

```diff
@@ -162,7 +162,6 @@ public static AudioItem resolveItem(Player player, String input) throws IllegalA
                 if (e.getMessage().contains("Vevo")) {
                     throw new IllegalStateException(Jsoup.clean(cause.getMessage(), Whitelist.none()), cause);
                 }
-                FlareBot.LOGGER.error(Markers.NO_ANNOUNCE, "Cannot get video '" + input + "'");
```

Comment:
How come?

Reply:
This get's logged to sentry unnecessarily, we don't get any debug info from it anyway. All errors are logged to the users in the channel.

Chose to silence the errors like this because 9/10 times it is something to do with youtube or connection and so it's not worth logging the events.

Reply:
But you aren't silencing? The point of this and then Markers.NO_ANNOUNCE is to silence it, if you run Sentry through our Filter then it will also handle correctly with the NO_ANOUNCE. I'm against this change

Reply:
Even if I did that, there is no point in logging the error itself, all it does it spam the logs with cannot get video with no debug info or anything like that.